fe/fe-core/src/main/java/com/starrocks/persist/CreateExternalCatalogOperationLog.java

```diff
@@ -0,0 +1,48 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Map;
+
+public class CreateExternalCatalogOperationLog implements Writable {
```

Comment:
How about create catalog?
Later when we support create internal catalog, we can use it too.
`CreateCatalogLog` is enough